website/src/app/blog/apr-2024-update/readme.mdx

```diff
@@ -0,0 +1,245 @@
+the control plane and Rust for the data plane.
+
+Why?
+
+Elixir has been getting lots of acclaim in recent years for its concurrency
+model and fault-tolerance features. And for good reason: it runs on Erlang's
+BEAM VM, the same technology that powers the telecom industry's most reliable
+systems. There's a good chance the device you're reading this on has an IP
+address handed out by an Erlang-powered telecom switch.
+
+As it turns out, managing connections for a remote access product is _a lot_
+like managing messages across a telecom network:
+
+```
+1. Peer A wants to connect to Peer B.
+2. Is it allowed?
+  Yes: here are their addresses and keys to secure the connection.
+  No: drop the connection.
+```
+
+And Elixir's concurrency model makes it easy to manage thousands of these
+connection "intents" on very little hardware -- just a few tiny VMs orchestrate
+all connections across all our customers, globally.
+
+And what about the data plane? For that, we turned to Rust.
+
+Rust forms the network backbone of Firezone, handling all the heavy lifting of
+encrypting and decrypting packets as they flow between Clients and Gateways. As
+far as systems languages go, Rust couldn't be a better fit for the job. Its
+memory safety guarantees eliminate entire classes of bugs that plague other
+systems languages, making it a great choice for a security-critical application
+like Firezone.
+
+And it has build targets for just about every platform under the sun. Our
+[core connectivity library](https://github.com/firezone/firezone/tree/main/rust/connlib),
+for example, runs reliably on iOS, Android, Windows, Linux, and macOS.
+
+We'll be sharing more about our stack choices in future blog posts, but suffice
+to say, we're very happy with the results so far.
+
+### What's unique about Firezone?
+
+There are a lot of remote access solutions out there, so what makes Firezone
+different?
+
+For starters, Firezone uses [WireGuard®](https://www.wireguard.com/) under the
+hood -- a new VPN protocol that's
+[faster](https://www.wireguard.com/performance) and
+[more secure](https://www.wireguard.com/formal-verification/) than traditional
+VPNs. But that's just the start.
+
+We learned from Firezone 0.x that organizations grappling with remote access at
+scale needed things like integrations with identity providers that keep
+directory information in sync, high availability features, and an easier way to
+manage access policies that don't require a PhD in network security.
+
+Firezone 1.0 delivers on all of that and more.
+
+#### High availability
+
+One major feature in 1.0 we want to mention specifically is high availability.
+
+Each Firezone Gateway is a tiny, self-contained binary that needs
+[only a single environment](/kb/deploy/gateways) variable to function. Throw it
+in VM, a container, on an IoT device -- it's lightweight enough to run
+everywhere. Its sole purpose is to shuttle encrypted packets between Clients and
+protected resources.
```

Comment:
I noticed you capitalized `Clients`, did you want to capitalize `Resources` too?

Reply:
Yeah, I think I'll remove `protected` so that it's a proper noun in Firezone parlance.
Otherwise, I've been keeping `protected resources` lowercase because that's more of a generic term I think.

Reply:
Makes sense